models/ac_generator.h

```diff
@@ -77,6 +77,13 @@ class ac_generator : public Node
 
   void get_status( DictionaryDatum& ) const;
   void set_status( const DictionaryDatum& );
+  using Node::sends_signal;
+
+  SignalType
+  sends_signal() const
+  {
+    return ALL;
+  }
```

Comment:
@tobikausk This works, but for the wrong reasons. For semantic consistency, `sends_signal()` has to return a value that represents the type of signal sent by a node. For the `ac_generator`, that would be `CONTINUOUS` or, to be even more precise, `STEPWISE_CONSTANT`. Thus, we should extend the `SignalType` enumeration in [`nest_types.h`](https://github.com/nest/nest-simulator/blob/48ca1d7090d8e0953c8e752425407a4c3b52d4b1/nestkernel/nest_types.h#L198) and adjust code correspondingly. `ALL` should only be used on the receiver side: A node may be able to handle any kind of input, but it will never send more than one type of signal.

Reply:
@tobikausk: please change `ALL` to the new constant `STEPWISE_CONSTANT` as @heplesser pointed out.